--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,12 +1,13 @@
 # SPDX-License-Identifier: MIT OR Apache-2.0
 
 [build-system]
-requires = ["hatchling", "hatch-vcs", "hatch-fancy-pypi-readme>=22.8.0"]
+requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 
 [project]
-dynamic = ["readme", "version"]
+readme = "README.md"
+version = "26.1.0"
 name = "structlog"
 description = "Structured Logging for Python"
 authors = [{ name = "Hynek Schlawack", email = "hs@ox.cx" }]
@@ -75,10 +76,6 @@ dev = [
 ]
 
 
-[tool.hatch.version]
-source = "vcs"
-raw-options = { local_scheme = "no-local-version" }
-
 
 [tool.pytest.ini_options]
 addopts = ["--strict-markers", "--strict-config", "--import-mode=importlib"]
